### ISSUE TITLE
Memory Meryl & estMerThresh

### ISSUE THREAD

**Issue by fmarletaz** [marbl/canu#1105]
Hi - Thanks for your continuing support. I am working on the assembly of a large polymorphic animal genome (2.5Gb). I have 165Gb of sequel data as input. The assembly crashed at the Meryl stage. There is no log. Here is the output of the `meryl.4341468_1.out` in `correction/0-mercounts`. Any suggestions? 

```
  Releasing 128MB from counting the size of each bucket.
 Allocating 8579MB for mer storage (7 bits wide).
 Filling mers into list:   10272.86 Mmers --  2.72 Mmers/second
/var/spool/slurmd/job4341468/slurm_script: line 29: 25532 Killed                  $bin/meryl -B -C -L 2 -v -m 16 -threads 16 -memory 98304 -s ../Skate.gkpStore -o ./Skate.ms16.WORKING
/var/spool/slurmd/job4341468/slurm_script: line 58: 29063 Segmentation fault      (core dumped) $bin/estimate-mer-threshold -h ./Skate.ms16.histogram -c 41 > ./Skate.ms16.estMerThresh.out.WORKING 2> ./Skate.ms16.estMerThresh.err
```
Another question, unless I missed it, there is no option to set the gridIOptions of the meryl stage, while it would benefit from it, as many systems have a specific queue (or partition) for large memory job. 

Thanks!

**Comment by skoren** [comment]
It looks like the job was killed for exceeding memory. It looks like it's trying to use 100gb of ram. How much did this job actually use when it was killed and how much did it request from  your grid (meryl.jobSubmit-01.sh would have that info)?

You can just try increasing the memory for meryl, there is a ```gridOptionsmeryl``` which you can set to request 150gb or 200 if you have it available. You can always get a list of Canu options by running ```canu -options``` and grid options with ```canu -options |grep gridOptions```.

**Comment by fmarletaz** [comment]
Hi - I have a slightly different question. For the same assembly, my trimming overlap stage was taking forever, and I saw this over post the issue #1014 and applied the approach you suggested successfully. I am worried I am running in the same problem at the unitigging step as well. If I set a threshold of 500, is that very conservation, how does setting arbitrarily a threshold would impact the assembly? Thanks!
